langchain4j-mistral-ai/src/main/java/dev/langchain4j/model/mistralai/internal/api/MistralAiChatCompletionChoice.java

```diff
@@ -20,4 +17,118 @@ public class MistralAiChatCompletionChoice {
     private MistralAiDeltaMessage delta;
     private String finishReason;
     private MistralAiUsage usage; // usageInfo is returned only when the prompt is finished in stream mode
+
+    public MistralAiChatCompletionChoice() {
+    }
+
+    public Integer getIndex() {
+        return this.index;
+    }
+
+    public MistralAiChatMessage getMessage() {
+        return this.message;
+    }
+
+    public MistralAiDeltaMessage getDelta() {
+        return this.delta;
+    }
+
+    public String getFinishReason() {
+        return this.finishReason;
+    }
+
+    public MistralAiUsage getUsage() {
+        return this.usage;
+    }
+
+    public void setIndex(final Integer index) {
+        this.index = index;
+    }
+
+    public void setMessage(final MistralAiChatMessage message) {
+        this.message = message;
+    }
+
+    public void setDelta(final MistralAiDeltaMessage delta) {
+        this.delta = delta;
+    }
+
+    public void setFinishReason(final String finishReason) {
+        this.finishReason = finishReason;
+    }
+
+    public void setUsage(final MistralAiUsage usage) {
+        this.usage = usage;
+    }
+
+    @java.lang.Override
+    public boolean equals(final java.lang.Object o) {
```

Comment:
Here is a good example of an `equals` method implementation, please follow this style:

```java
    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        Audio that = (Audio) o;
        return Objects.equals(this.url, that.url)
            && Objects.equals(this.base64Data, that.base64Data)
            && Objects.equals(this.mimeType, that.mimeType);
    }
```

Reply:
is inlining a requirement ?

Reply:
What do you mean?

Reply:
if (this == o) return true;
if (o == null || getClass() != o.getClass()) return false;
instead of 
if (this == obj) {
            return true;
        }
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }

Reply:
Not really, but it would be nice to keep the same style if there is no reason to not to